tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -82,28 +84,91 @@ async def test_create_sensors(
     assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.TOTAL
     assert state.attributes.get(ATTR_DEVICE_CLASS) == SensorDeviceClass.ENERGY
 
+    state = hass.states.get("sensor.heat_meter_heat_usage")
+    assert not state
+
+    state = hass.states.get("sensor.heat_meter_heat_previous_year_gj")
+    assert state
+    assert state.state == "111.0"
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == UnitOfEnergy.GIGA_JOULE
+
+    state = hass.states.get("sensor.heat_meter_heat_previous_year")
+    assert not state
+
     state = hass.states.get("sensor.heat_meter_volume_usage")
     assert state
     assert state.state == "456.0"
     assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == UnitOfVolume.CUBIC_METERS
     assert state.attributes.get(ATTR_STATE_CLASS) == SensorStateClass.TOTAL
 
+    entity_reg = er.async_get(hass)
+
     state = hass.states.get("sensor.heat_meter_device_number")
     assert state
     assert state.state == "devicenr_789"
     assert state.attributes.get(ATTR_STATE_CLASS) is None
-    entity_registry_entry = entity_registry.async_get("sensor.heat_meter_device_number")
+    entity_registry_entry = entity_reg.async_get("sensor.heat_meter_device_number")
```

Comment:
I think this change is not needed - maybe a bad rebase?

Reply:
Yes, thanks. Fixed this.